scripts/run_frontend_tests.py

```diff
@@ -113,22 +113,19 @@ def main(args=None):
         'on your filesystem.',
         'Running test in development environment'])
 
+    cmd = [
```

Comment:
For reviewers: In the last 2 months the frontend test size has increased 2.5x times which lead to increase of space requirement, I've increased the memory size! Though I also suspect some memory leakage in our tests may be because of some structure of angularJs/angular! I'll create an issue for now and will look into it after migration!

Should I add a code-comment explaining why we need `max-old-space-size` flag?

Reply:
Issue at #13679